main/Helper.cpp

```diff
@@ -526,50 +526,54 @@ std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, i
 	return ret;
 }
 
-//convert date string 10/12/2014 10:45:58 en  struct tm
-void DateAsciiTotmTime (std::string &sTime , struct tm &tmTime  )
+std::string TimeToString(time_t ltime, _eTimeFormat format)
```

Comment:
Should ltime not be a pointer ? (time expects a pointer, also localtime)
Also makes it a bit more consistant
You could also have choosen for using the strftime format...